test/Table_Tests/src/IO/Read_Many_Spec.enso

```diff
@@ -447,33 +447,8 @@ add_specs suite_builder =
                 r3.at "Value" . to_vector . should_equal ["Y"]
                 r3.at "Value 1" . first . should_be_a Table
 
-private with_temp_dir callback =
-    base_dir = enso_project.data / "transient" / "read_many_test"
+private with_temp_dir test_name:Text callback =
+    base_dir = enso_project.data / "transient" / test_name
```

Comment:
- won't the problems go away if 524405f4dca5f3d655bf058c8f0fbdd1f376088c stops using the same directory again and again?
- if not, feel free to remove 524405f4dca5f3d655bf058c8f0fbdd1f376088c with `git reset HEAD~1; git push -f`

Reply:
Thanks. I think this is a much more elegant solution. And yes, the problem should go away. Let's merge.